src/install_kcov.sh

```diff
@@ -9,23 +9,32 @@ GITHUB_KCOV="https://api.github.com/repos/SimonKagstrom/kcov/releases/latest"
 # Fall back to $KCOV_DEFAULT_VERSION from the kcov archive if the latest is unavailable.
 
 KCOV_VERSION=$(curl -s ${GITHUB_KCOV} | jq -Mr .tag_name || echo)
-KCOV_VERSION=${KCOV_VERSION+$KCOV_DEFAULT_VERSION}
+KCOV_VERSION=${KCOV_VERSION:-$KCOV_DEFAULT_VERSION}
 
-KCOV_TGZ="https://github.com/SimonKagstrom/kcov/archive/${KCOV_DEFAULT_VERSION}.tar.gz"
+KCOV_TGZ="https://github.com/SimonKagstrom/kcov/archive/${KCOV_VERSION}.tar.gz"
 
 rm -rf kcov-${KCOV_VERSION}/
 mkdir kcov-${KCOV_VERSION}
-curl -L "${KCOV_TGZ}" | tar xzvf - -C kcov-${KCOV_VERSION} --strip-components 1
+curl -L --retry 3 "${KCOV_TGZ}" | tar xzvf - -C kcov-${KCOV_VERSION} --strip-components 1
+
+num_proc=1
+if [ "${PARALLEL_BUILD:-}" != "" ]; then
+    if [ "$(uname -s)" = "Darwin" ]; then
```

Comment:
Please directly check if `nproc`/`sysctl` is present instead OS sniffing. There is no guarantee that `nproc` is present outside of a Mac.